aptos-move/aptos-gas-calibration/samples_ir/borrow/freeze_ref.mvir

```diff
@@ -0,0 +1,46 @@
+module 0xcafe.FreezeRef {
```

Comment:
Are these `*.mvir` files some test fixtures to be maintained, or are they generated?

Reply:
I don't think these are generated, but are created by the aptos-move team. These files are also present in the upstream on their remote repo, so we should have parity there, for example: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas-calibration/samples_ir/borrow/freeze_ref.mvir

Same with all the other `.mvir` files.